002-vm-boot-from-iso/Makefile

```diff
@@ -0,0 +1,45 @@
+vmname="Haiku"
+iso=haiku-r1alpha4.iso
+remote_iso=/ipfs/QmW9on9UeKCVDb5dsTEZJQwQoH53gXUtVfWvSBhf2Thmyq/isos/haiku-r1alpha4.1-iso.tar.xz
+
+ifeq (, $(shell which VBoxManage))
+  $(error "No VBoxManage found in $(PATH). Please install VirtualBox: https://www.virtualbox.org/wiki/Downloads")
+endif
+
+define notfound
+$(iso) not found
+Make sure the daemon is running and mounted:
+	ipfs daemon &
+	sleep 10
+	ipfs mount
+
+If you are having trouble mounting IPFS, please see https://github.com/jbenet/go-ipfs/blob/master/docs/fuse.md
+endef
+export notfound
```

Comment:
as the makefiles get more complex, would be nice to be able to refer to these pieces of test from various places (so we can update them once). maybe this is text we can put inside files in a `000-common` dir or something. 

the downside is it's less self contained :( i like the directories are basically independent. **shrug** thoughts?